features/networking/service.feature

```diff
@@ -154,3 +154,44 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-11645
+  @admin
+  Scenario: Create loadbalancer service
+    Given I have a project
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/e3e88a1b3706a7ede5c6cb7805fa72d3218a9998/networking/ping_for_pod_containerPort.json |
+    Then the step should succeed
+
+    # Create loadbalancer service
+    When I run the :create_service_loadbalancer client command with:
+      | name | hello-pod |
+      | tcp  | 5678:8080       |
+    Then the step should succeed
+
+    # Get the external ip of the loadbaclancer service
+    And I wait up to 20 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
+      | resource | svc |
+      | resource_name | hello-pod |
+      | template | {{(index .status.loadBalancer.ingress 0).hostname}} |
```

Comment:
@huiran0826 log error showed on this step:

"Error executing template: template: output:1:3: executing "output" at <index .status.loadBalancer.ingress 0>: error calling index: index of untyped nil. Printing more information for debugging the template:
      	template was:
      		{{(index .status.loadBalancer.ingress 0).hostname}}
      	raw data was:
      		{"apiVersion":"v1","kind":"Service","metadata":{"creationTimestamp":"2020-02-10T09:40:09Z","finalizers":["service.kubernetes.io/load-balancer-cleanup"],"labels":{"app":"hello-pod"},"name":"hello-pod","namespace":"um-0h","resourceVersion":"141971","selfLink":"/api/v1/namespaces/um-0h/services/hello-pod","uid":"06a09f6b-603c-46ce-acd6-48871c8d49da"},"spec":{"clusterIP":"172.30.235.169","externalTrafficPolicy":"Cluster","ports":[{"name":"5678-8080","nodePort":32572,"port":5678,"protocol":"TCP","targetPort":8080}],"selector":{"app":"hello-pod"},"sessionAffinity":"None","type":"LoadBalancer"},"status":{"loadBalancer":{}}}
      	object given to template engine was:
      		map[apiVersion:v1 kind:Service metadata:map[creationTimestamp:2020-02-10T09:40:09Z finalizers:[service.kubernetes.io/load-balancer-cleanup] labels:map[app:hello-pod] name:hello-pod namespace:um-0h resourceVersion:141971 selfLink:/api/v1/namespaces/um-0h/services/hello-pod uid:06a09f6b-603c-46ce-acd6-48871c8d49da] spec:map[clusterIP:172.30.235.169 externalTrafficPolicy:Cluster ports:[map[name:5678-8080 nodePort:32572 port:5678 protocol:TCP targetPort:8080]] selector:map[app:hello-pod] sessionAffinity:None type:LoadBalancer] status:map[loadBalancer:map[]]]
      STDERR:
      error: error executing template "{{(index .status.loadBalancer.ingress 0).hostname}}": template: output:1:3: executing "output" at <index .status.loadBalancer.ingress 0>: error calling index: index of untyped nil"

Reply:
@weliang1 The error because after create loadbalancer svc, it needs some time to take effect, so you may see from errors, the loadBalancer is empty.  That is why I add steps take 20s to let the step pass.            {"apiVersion":"v1","kind":"Service","metadata":{"creationTimestamp":"2020-02-10T09:40:09Z","finalizers":["service.kubernetes.io/load-balancer-cleanup"],"labels":{"app":"hello-pod"},"name":"hello-pod","namespace":"um-0h","resourceVersion":"141971","selfLink":"/api/v1/namespaces/um-0h/services/hello-pod","uid":"06a09f6b-603c-46ce-acd6-48871c8d49da"},"spec":{"clusterIP":"172.30.235.169","externalTrafficPolicy":"Cluster","ports":[{"name":"5678-8080","nodePort":32572,"port":5678,"protocol":"TCP","targetPort":8080}],"selector":{"app":"hello-pod"},"sessionAffinity":"None","type":"LoadBalancer"},"status":{"loadBalancer":{}}}

The next loop it can get correct result.
 oc get svc hello-pod --template=\{\{\(index\ .status.loadBalancer.ingress\ 0\).hostname\}\} --config=/home/jenkins/workspace/Runner-v3/workdir/ocp4_testuser-43.kubeconfig
      a06a09f6b603c46ceacd648871c8d49d-539528633.us-east-2.elb.amazonaws.com